plugins/star-rating/frontend/public/javascripts/countly.views.js

```diff
@@ -1,2320 +1,363 @@
-            }
-            $("#ratings_version_" + self._tab).clySelectSetSelection("", "");
-            $("#ratings_version_" + self._tab + " .text").html('<div class="placeholder" data-localize="feedback.select-version">' + jQuery.i18n.map['feedback.select-version'] + '</div>');
-            self.loadVersionData();
-        });
-    },
-    /**
-     * This is for render platform dropdown select view.
-     * @namespace starView
-     * @method loadPlatformData
-     */
-    loadWidgetData: function() {
-        $(".widget-list").html('<div data-value="All Widgets" class="widget-option item" data-localize="star.all-widgets">' + jQuery.i18n.map['star.all-widgets'] + '</div>');
-        this.templateData.widget.sort();
-        for (var i = 0; i < this.templateData.widget.length; i++) {
-            $(".widget-list").append('<div data-value="' + this.templateData.widget[i]._id + '" class="widget-option item" data-localize="">' + this.templateData.widget[i].popup_header_text + '</div>');
-        }
-    },
-    /**
-     * This is for render rating dropdown select view.
-     * @namespace starView
-     * @method loadRatingData
-     * @param {boolean} keepOpen - will it keep open?
-     */
-    resetFilterBox: function(keepOpen) {
-        var self = this;
-        var values = this.ratingFilter[this._tab];
-        if (!keepOpen) {
-            $("#rating-selector").removeClass('active');
-            $("#rating-selector-graph").removeClass('active');
-            $(".star-rating-selector-form").hide();
-        }
-        if (values.rating === "") {
-            $("#ratings_rating_" + this._tab).clySelectSetSelection("", "");
-            $("#ratings_rating_" + this._tab + " .text").html('<div class="placeholder" data-localize="feedback.select-rating">' + jQuery.i18n.map['feedback.select-rating'] + '</div>');
-        }
-        else {
-            $("#ratings_rating_" + this._tab).clySelectSetSelection(values.rating, jQuery.i18n.map[this.localizeStars[parseInt(values.rating) - 1]]);
-        }
-
-        if (values.platform === "") {
-            $("#ratings_platform_" + this._tab).clySelectSetSelection("", "");
-            $("#ratings_platform_" + this._tab + " .text").html('<div class="placeholder" data-localize="feedback.select-platform">' + jQuery.i18n.map['feedback.select-platform'] + '</div>');
-        }
-        else {
-            $("#ratings_platform_" + this._tab).clySelectSetSelection(values.platform, values.platform);
-        }
-
-        if (values.version === "") {
-            $("#ratings_version_" + this._tab).clySelectSetSelection("", "");
-            $("#ratings_version_" + this._tab + " .text").html('<div class="placeholder" data-localize="feedback.select-version">' + jQuery.i18n.map['feedback.select-version'] + '</div>');
-        }
-        else {
-            $("#ratings_version_" + this._tab).clySelectSetSelection(values.version, values.version.replace(/:/g, "."));
+var FEATURE_NAME = 'star-rating';
```

Comment:
please wrap everything in this file within an anonymous function
(function(){
...
}());